server/core/src/json_serialization.cpp

```diff
@@ -15,6 +15,8 @@ namespace
     {
         auto* s = parseMspForStr(&_p);
 
+        // TODO Returning nullptr isn't incorrect. However, parseMspForStr() will return
+        // a nullptr if the string value is "null". This feels incorrect.
         if (!s) {
             THROW(SYS_INVALID_INPUT_PARAM, "Failed to convert microservice argument to string.");
         }
```

Comment:
Handle TODO

`nullptr` in this case can mean success or failure.

Reply:
Created separate issue for this. See #6810.

This will be handled by a different PR.

Reply:
Removed comment now that it's captured in an issue.